x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/model_select.test.tsx

```diff
@@ -98,6 +98,23 @@ describe('ModelSelect', () => {
       })
     );
   });
+  it('sets placeholder flag on selecting a placeholder item', () => {
+    setMockValues(DEFAULT_VALUES);
+
+    const wrapper = shallow(<ModelSelect />);
```

Comment:
We are[ moving from](https://elastic.slack.com/archives/CA65B8NUR/p1701185411645739) **enzyme** to **react-testing-library**. Is it possible to convert these tests to ones compitible with **react-testing-library**?

Reply:
@saikatsarkar056 Thanks for the heads up, I just learned about this.
Since this is a critical usability fix that must go into 8.12, are you OK getting this PR merged as-is and deferring the rewriting of tests (possibly with a larger scope) to a tech debt ticket?